query/security/embeddedldap/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <bean id="installEmbeddedLdap" class="org.codice.ddf.admin.ldap.embedded.InstallEmbeddedLdap">
+        <argument ref="configuratorFactory"/>
+        <argument ref="featureActions"/>
+    </bean>
+
+    <service ref="installEmbeddedLdap" interface="org.codice.ddf.admin.api.fields.FunctionField">
+        <service-properties>
+            <entry key="id" value="installEmbeddedLdap"/>
```

Comment:
nit, we overload the word id. Call we call this fieldProviderId or something? should we do a name convention here as well? something like `ldap.installEmbeddedLdap`?